packages/core/admin/admin/src/pages/SettingsPage/pages/Webhooks/EditView/components/WebhookForm/tests/index.test.js

```diff
@@ -2,26 +2,41 @@ import { render, screen, waitFor, fireEvent } from '@testing-library/react';
 import React from 'react';
 import { Router } from 'react-router-dom';
 import { createMemoryHistory } from 'history';
+import { QueryClientProvider, QueryClient } from 'react-query';
 import { lightTheme, darkTheme } from '@strapi/design-system';
+import { NotificationsProvider } from '@strapi/helper-plugin';
+
 import en from '../../../../../../../../translations/en.json';
 import Theme from '../../../../../../../../components/Theme';
 import ThemeToggleProvider from '../../../../../../../../components/ThemeToggleProvider';
 import LanguageProvider from '../../../../../../../../components/LanguageProvider';
 import WebhookForm from '../index';
 
+jest.mock('../../../../../../../../hooks', () => ({
+  useThemeToggle: jest.fn(() => ({ currentTheme: 'light', themes: { light: lightTheme } })),
+}));
```

Comment:
I believe you can also get rid of this, if you wrap the test below into `ThemeProvider` instead of `ThemeToggleProvider`?

Reply:
Do you still need the mock now that `ThemeToggleProvider` is gone?